doc/source/programs/gdal_raster_proximity.rst

```diff
@@ -0,0 +1,81 @@
+.. _gdal_raster_proximity:
+
+================================================================================
+``gdal raster proximity``
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Produces a raster proximity map.
+
+.. Index:: gdal raster proximity
+
+Synopsis
+--------
+
+.. program-output:: gdal raster proximity --help-doc
+
+Description
+-----------
+
+:program:`gdal raster proximity` generates a raster proximity map indicating the distance from
+the center of each pixel to the center of the nearest pixel identified as a target pixel.
+Target pixels are those in the source raster for which the raster pixel value is in the set of
+target pixel values.
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_raster_create_copy.rst
+
+.. include:: gdal_options/co.rst
+.. include:: gdal_options/overwrite.rst
+
+.. include:: gdal_options/ot.rst
+
+.. option:: -b, --band <BAND>
+
+    Input band (1-based index)
+
+.. option:: --target-values <TARGET-VALUES>
```

Comment:
What is the format of the list? Are ranges allowed?

Reply:
No, it would be a nice addition though.

Reply:
I have clarified the list format.

Reply:
Using this in a pipeline after `gdal raster reclassify` would effectively allow ranges to be used as the target value.